client/src/layers/Kanban/index.tsx

```diff
@@ -18,13 +13,13 @@ const Kanban = () => {
 
   const todoList = storyList
     .filter((item) => item.status === 'TODO')
-    .sort((a, b) => a.order - b.order);
+    .sort((a, b) => Number(a.order) - Number(b.order));
   const onGoingList = storyList
     .filter((item) => item.status === 'IN_PROGRESS')
-    .sort((a, b) => a.order - b.order);
+    .sort((a, b) => Number(a.order) - Number(b.order));
   const finishList = storyList
     .filter((item) => item.status === 'DONE')
-    .sort((a, b) => a.order - b.order);
+    .sort((a, b) => Number(a.order) - Number(b.order));
```

Comment:
```suggestion
    .sort(storySortByOrder(a,b));
```
반복되는 정렬함수를 빼보는 건 어떨까요?? `utils/sort.ts` 가 준비되어있습니다.😉

Reply:
저도 찬성입니다! 다만.. sortStoryByOrder 로 해주시면 안될까요? 함수는 동사로 시작하는게 좋으니깐..